packages/manager/apps/container/src/container/nav-reshuffle/sidebar/Assistance/index.tsx

```diff
@@ -77,27 +80,46 @@ const AssistanceSidebar: React.FC<ComponentProps<AssistanceProps>> = ({
     trackingPlugin.trackClick({ name: `navbar_v3_entry_home::${id}`, type: 'navigation' });
   };
 
+  if (isShort) return (
+    <OsdsMenu className='w-full fixed z-[1000] left-[0.3rem] bottom-[3rem]' id="useful-links" role="menu">
+      <OsdsButton
+        slot='menu-title'
+        className='w-[4rem]'
+        color={ODS_THEME_COLOR_INTENT.primary}
+        variant={ODS_BUTTON_VARIANT.ghost}
+        size={ODS_BUTTON_SIZE.sm}
+        title={t('sidebar_assistance_title')}
+        contrasted
+      >
+        <OsdsIcon
+          name={ODS_ICON_NAME.ELLIPSIS}
+          contrasted
+        />
+      </OsdsButton>
+      {nodeTree.children.map((node: Node) => (
+        <OsdsMenuItem key={node.id}>
+          <ShortAssistanceLinkItem node={node} />
+        </OsdsMenuItem>
+      ))}
+    </OsdsMenu>
```

Comment:
Note that Menu has been deprecated in ODS 18 (and we will need to use a combo of popover and button to achieve this)